<!--
 Interface to the Arduino board.

 To interface with Thorp's Arduino Mega 2560 board, flash the provided firmware. Note that analog and
 trigger pins are hardcoded, so probably you need to adapt the first lines of the firmware to your needs.
 You must install the Arduino udev rule to make the board available on the default port "/dev/arduino":

  $ roscd thorp_boards
  $ sudo cp resources/58-arduino.rules /etc/udev/rules.d
 
 and then replug the USB cable.
-->
<launch>
  <node pkg="thorp_boards" type="arduino_node" name="arduino_node" output="screen" respawn="true">
    <rosparam file="$(find thorp_boards)/param/arduino.yaml" command="load"/>
  </node>
</launch>
